src/server/controllers/users.controller.js

```diff
@@ -95,7 +98,111 @@ export function getProfileById(req, res, next) {
   }
 }
 
-export function signin(req, res, next) {
+export function localSignUp(req, res, next) {
+  let cipher = crypto.createCipher('aes256', req.body.password);
+  cipher.update(req.body.email, 'ascii', 'hex');
+  let cryptoPassword = cipher.final('hex');
+
+  let registrationData = {
```

Comment:
How about just register email & password then move to edit general profile? Maybe it would be same flow with Facebook login.